smac/epm/epm_gpytorch/gaussian_process_gpytorch.py

```diff
@@ -0,0 +1,365 @@
+import typing
+
+import warnings
+from collections import OrderedDict
+
+import gpytorch
+import numpy as np
+import torch
+from botorch.optim.numpy_converter import module_to_array, set_params_with_array
+from botorch.optim.utils import _scipy_objective_and_grad
+from gpytorch.constraints.constraints import Interval
+from gpytorch.distributions.multivariate_normal import MultivariateNormal
+from gpytorch.kernels import Kernel
+from gpytorch.likelihoods import FixedNoiseGaussianLikelihood, GaussianLikelihood
+from gpytorch.means import ZeroMean
+from gpytorch.mlls import ExactMarginalLogLikelihood
+from gpytorch.models import ExactGP
+from gpytorch.priors import HorseshoePrior
+from gpytorch.utils.errors import NotPSDError
+from scipy import optimize
+
+from smac.configspace import ConfigurationSpace
+from smac.epm.base_gp import BaseModel
+from smac.utils.constants import VERY_SMALL_NUMBER
+
+warnings.filterwarnings("ignore", module="gpytorch")
+
+
+class ExactGPModel(ExactGP):
```

Comment:
in local_bo/utils.py you also create gp kernels. Would be nice to have it here only.